### PR TITLE
Add breaking change docs for authentication [release-2020-12]

### PR DESCRIPTION
**Pull request by mokimo**

### CODE REVIEW

File: releases/release-2020-12.md [hunk 1]
```diff
@@ -271,6 +270,66 @@ When you start the new server version, the publication index is empty. To use th
 - 2) If you want to deploy/start your new server version without a preparation step, you can index the most recent publication with an additional parameter `--since`. For example `livingdocs-server elasticsearch-index --handle=li-publications --since 1m -y` does indexing all publications published within the last month. As soon as you're done with this indexing step, you can run another background without `--since` argument to index all publications.
 
 
+## Authentication
+With the improved [authentication workflow](https://github.com/livingdocsIO/livingdocs-server/pull/3225), we have some additional breaking changes.
+
+🔥 third party applications / e2e-tests may need some adaptions correctly support cookies (*Note, see changes for local development to test correctly)
```
Comment (DaRaFF) [inline releases/release-2020-12.md:276]: - support -> supporting
- remove *

File: releases/release-2020-12.md [hunk 2]
```diff
@@ -271,6 +270,66 @@ When you start the new server version, the publication index is empty. To use th
...
+
+🔥 Local development on chrome now requires a SSL setup for authentication to work. Setting up [a certificate locally](https://github.com/livingdocsIO/livingdocs-editor/blob/master/config/cert.js) and proxying requests using the [editor environment](https://github.com/livingdocsIO/livingdocs-editor/blob/master/config/environments/local.js#L14) config is advised.
```
Comment (DaRaFF) [inline releases/release-2020-12.md:278]: chrome -> Chrome

Comment (DaRaFF) [reply releases/release-2020-12.md:278]: The local certificate documentation should also be added to the business doc for local development -> https://docs.livingdocs.io/evaluation-guide/getting-started-with-local-development

**Review by DaRaFF — APPROVED** [review]

File: releases/release-2020-12.md (hunk 2 above)
Comment (mokimo) [reply releases/release-2020-12.md]: https://github.com/livingdocsIO/livingdocs/pull/355

**Pull request merged**